### DISCUSSION TITLE
Issue observed while using Background Keyword in feature file

### DISCUSSION THREAD

**Discussion by suumitru01** [cypress-io/cypress#26297] [Component Testing]
Hi Team ,
I working on one sample project where I am using cypress version of 9.6.0 and cypress-cucumber-preprocessor version as 4.3.1 and issue which I am facing is whenever I tried using Background keyword in feature file to run the common steps under different scenarios at the end of the execution getting one error  saying that 

<img width="312" alt="image" src="https://user-images.githubusercontent.com/65500546/228719939-bb43593b-87e0-47a3-97c3-b26c13920af0.png">

Could you please suggestion solution same , That will be helpful ,Thanks

**Comment by jjhelguero** [comment]
This is more related to [cypress-cucumber-preprocessor](https://github.com/badeball/cypress-cucumber-preprocessor) and might be better to post your question on that repo to get an answer.
